packages/backend/src/graphql/schema.graphql

```diff
@@ -2,6 +2,14 @@ type Query {
   "Fetch an asset by its ID."
   asset("Unique identifier of the asset." id: String!): Asset
 
+  "Get an asset based on its currency code and scale if it exists."
+  assetByCodeAndScale(
```

Comment:
we should add bruno requests for these queries

Reply:
An asset code and scale combination is unique, right? Might make sense for the graphql schema to reflect that in some way.

Reply:
I thought these queries might be reused in the future, because they are searching by the unique column(s) of the tables. But if not, I can just filter everything in the script and it might be a bit faster than just querying the db that much.

Reply:
> An asset code and scale combination is unique, right?

Yes, has a unique constraint in DB. 

@njlie what do you mean by "Might make sense for the graphql schema to reflect that in some way."?

Reply:
What I mean to say is, from the point of view of someone integrating with the GraphQL API and looking at it for the first time, seeing a resolver like `assetByCodeAndScale` would be an effective way to communicate to that audience that those two fields together is a unique combination

Reply:
Ah, I see what you mean. As in, the schema itself reflects to the integrator about how assets work. I thought you meant we needed to add a comment or something to the schema. Understand now.

Reply:
I agree in general, let's keep these resolvers. @oana-lolea I think the outstanding item is just to remove some lines in the resolver test file